### DISCUSSION TITLE
Old icon associated files recovery

### DISCUSSION THREAD

**Discussion by tom-vb** [d2phap/ImageGlass#1308] [Q&A] [answered]
Hello, I don't know what happened but I was happy and visually adapted to the blue icon of files associated with Imageglass but suddenly I used another version and now the icons have changed. The question is, how can recover (if possible) the old icon of files associated with Imageglass? see attached.

thanks

![20220304_180752_ksnp](https://user-images.githubusercontent.com/83949165/156809232-32ccdf60-5e25-43e7-a2ae-c9331370b1d3.png)
.

**Comment by d2phap** [comment, marked as answer]
You delete all icons in Ext-Icons folder of ImageGlass, then open Settings > Set as default photo viewer button again.

**Reply by tom-vb** [reply]
worked, many thanks!
